control-plane/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/go-logr/logr v0.4.0
 	github.com/google/go-cmp v0.5.7
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
-	github.com/hashicorp/consul/api v1.14.0
+	github.com/hashicorp/consul/api v1.10.1-0.20220822180451-60c82757ea35
```

Comment:
I think this should point to the 1.14 API package?

Reply:
@david-yu , thanks for the feedback. This PR depends on the change in consul after [api/v1.14.0](https://github.com/hashicorp/consul/releases/tag/api%2Fv1.14.0). So shall I create a new tag for consul/api?

Reply:
I think it is ok not to create a tag! We do use the above format to include unreleased API changes. It is strange only because golang interprets it as some `1.10.1-*` version even though it contains all the 1.14 changes but that has been true for sometime now.